src/common/defines.h

```diff
@@ -0,0 +1,33 @@
+/**
+ *  Defines for your entire project at one place
+ *
+ *	@author 	Tilen Majerle
+ *	@email		tilen@majerle.eu
+ *	@website	http://stm32f4-discovery.com
+ *	@version 	v1.0
+ *	@ide		Keil uVision 5
+ *	@license	GNU GPL v3
+ *
+ * |----------------------------------------------------------------------
+ * | Copyright (C) Tilen Majerle, 2014
+ * |
+ * | This program is free software: you can redistribute it and/or modify
+ * | it under the terms of the GNU General Public License as published by
+ * | the Free Software Foundation, either version 3 of the License, or
+ * | any later version.
+ * |
+ * | This program is distributed in the hope that it will be useful,
+ * | but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * | MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * | GNU General Public License for more details.
+ * |
+ * | You should have received a copy of the GNU General Public License
+ * | along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ * |----------------------------------------------------------------------
+ */
+#ifndef TM_DEFINES_H
+#define TM_DEFINES_H
+
+/* Put your global defines for all libraries here used in your project */
+
+#endif
```

Comment:
What is this file for?

Is it required by some 3rd party code to have a `defines.h` configuration header file?
=> Then place the file to `include/<library>/defines.h` with proper documentation.

Is it used by our code only? Then consider removing it.

Reply:
vyžadováno 'modulem' pro výpočet CRC (aktuálně přesunuto do "lib/CRC")